core-plugins/src/main/java/io/cdap/plugin/batch/joiner/Joiner.java

```diff
@@ -134,6 +140,13 @@ public JoinDefinition define(AutoJoinerContext context) {
           case OUTPUT_SCHEMA:
             OutputSchemaError schemaError = (OutputSchemaError) error;
             failure.withOutputSchemaField(schemaError.getField());
+            break;
+          case DISTRIBUTION_SIZE:
+            failure.withConfigProperty(JoinerConfig.DISTRIBUTION_FACTOR);
+            break;
+          case DISTRIBUTION_STAGE:
```

Comment:
add one case for the broadcast error

Reply:
whoops, forgot to push my changes..its there now